docs/4.2/admin-guide.md

```diff
@@ -34,7 +34,7 @@ $ sudo mkdir -p /var/lib/teleport
 
 ### Teleport Checksum
 
-Gravitational Teleport provides a checksum from the 
+Gravitational Teleport provides a checksum from the
```

Comment:
It's a bit of pain / makes it harder to review when removing trailing spaces, but I think it's a good practice and something we should write as part of our docs checks.

Reply:
Ooops, didn't realize how many diffs that produced.
Reverted.